### DISCUSSION TITLE
Privacy concerns

### DISCUSSION THREAD

**Discussion by wickles** [kangyu-california/PersistentWindows#211] [Q&A] [answered]
Could somebody share a quick explanation of privacy risks associated with using this software? Does it capture images of the screen in order to function, or are the "snapshots" based purely on windows' coordinates and dimensions?

**Comment by kangyu-california** [comment, marked as answer]
I have added Privacy Statement section in [readme](https://github.com/kangyu-california/PersistentWindows)

**Comment by starfoxbeta** [comment]
Hey @kangyu-california  awesome app.  But can you provide assurance within the privacy statement that says that this app does not transmit data from what looks to be essentially a keylogger built into this?  Why do you need to capture keystrokes?  I'm curious why that's technically required to solve for this use case.  Thank you.  cc @wickles

**Comment by kangyu-california** [comment]
PW only reads keyboard event from PW icon window or webpage commander window, and interpret the one letter keystroke as command shortcut to capture/restore window snapshot or navigate though tabs or webpage in web browsers. It seems to me the command shortcuts are more  convenient than traditional hotkeys. For more details please read online help.

All window layout related data are stored within C:\Users\\[User]\AppData\Local\PersistentWindows, and PW does not transmit any user input/data to any server.
